### PR TITLE
Update nightly

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

cc @timkpaine

### CODE REVIEW

**Comment by timkpaine** [comment]
@xhochy trying to fix the deviation from `main` with the same version first in https://github.com/conda-forge/rust-feedstock/pull/103

**Comment by timkpaine** [comment]
ideally `dev` would be strictly ahead of `main` at all times, with just merge commits and version increments

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-{% set version = "1.56.0" %}
-{% set year = "2021" %}
-{% set month = "08" %}
-{% set day = "16" %}
+{% set version = "1.60.0" %}
+{% set year = "2022" %}
+{% set month = "04" %}
```
Comment (timkpaine) [inline recipe/meta.yaml:3]: `05`? 😆 time flies

Comment (xhochy) [reply recipe/meta.yaml:3]: This is the same version as in the `polars` PR.

**Comment by timkpaine** [comment]
`nightly` includes today so it looks like `05-15` is already available https://static.rust-lang.org/dist/2022-05-15/

File: recipe/meta.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ source:
   url: https://static.rust-lang.org/dist/{{ year }}-{{ month }}-{{ day }}/rust-nightly-aarch64-apple-darwin.tar.gz  # [osx and arm64]
   url: https://static.rust-lang.org/dist/{{ year }}-{{ month }}-{{ day }}/rust-nightly-x86_64-pc-windows-msvc.tar.gz  # [win64]
   sha256: 29e337c998149cb7f506ef3ebbbdd36a89885589cd27c54f2f31a4a844c857fc  # [win64]
-  sha256: b3607d66c68b2551dca122246a1bcbd47ec08dee5c46d1a60d41f356a1355b48  # [osx and x86_64]
+  sha256: 06caee68243a8c56a6b4f3dc072a8795549fd09a8dda4899c68dda1609e0539c  # [osx and x86_64]
   sha256: db3e2bff208bd596a59ccc5dc86d0aff2051fd671b804c06f5973994374a021f  # [linux and x86_64]
   sha256: 9f2f30f915ddac30054e587df3bb39b997c3857f2dd4547c60b5271a713bf257  # [linux and aarch64]
   sha256: fb01bc7fb19d34b2a0ba1a26f8df80e2dc0f07e8d2d77d409f543ca207f39497  # [linux and ppc64le]
```
Comment (timkpaine) [inline recipe/meta.yaml:22]: reset build number to 0

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (timkpaine) [reply recipe/meta.yaml:3]: 👍

**Comment by Maxyme** [comment]
@xhochy thanks for updating the nightly version!

**Comment by xhochy** [comment]
@conda-forge/rust This is ready for review.

**Pull request merged**